types/svgo/tsconfig.json

```diff
@@ -14,10 +14,12 @@
         ],
         "types": [],
         "noEmit": true,
-        "forceConsistentCasingInFileNames": true
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true,
+        "allowSyntheticDefaultImports": true
```

Comment:
Is this actually required? Other modules just have `esModuleInterop` e.g. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/67ff0213632be98f732e9c95d3812d069affe174/types/lodash/scripts/tsconfig.json#L6

Reply:
It is indeed sufficient for TS 2.7+, but the lint set includes running tests for TS 2.6, where I ran into

```
ERROR: 1:8    expect  Compile error in typescript@2.6 but not in typescript@2.7.
Fix with a comment '// TypeScript Version: 2.7' just under the header.
Module '".../DefinitelyTyped/types/svgo/index"' has no default export.
```

thus added the flag.

Reply:
Hm. OK. `allowSyntheticDefaultImports` alone is also sufficient, so you can remove `esModuleInterop`. :P